english/marshmallow.md

```diff
@@ -941,6 +941,6 @@ Click [here](https://en.wikipedia.org/wiki/Acupuncture) to learn about Accupunct
 
 As you approach the [Stargate](stargate/GateDial.md) with trepidation, you can hear the troops getting closer.
 
-Health and Life Lovely [Quotes](./health/health.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.